DMX.Portal.Web/Views/Pages/_Host.cshtml

```diff
@@ -11,6 +11,8 @@
     <base href="~/" />
     <link rel="stylesheet" href="css/bootstrap/bootstrap.min.css" />
     <link href="css/site.css" rel="stylesheet" />
+    <link href="_content/Syncfusion.Blazor.Themes/bootstrap5.css" rel="stylesheet" />
```

Comment:
We should use fabric.css

So it should be:

`<link href="_content/Syncfusion.Blazor.Themes/fabric.css" rel="stylesheet" />`